bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/html/ScheduleDocument.java

```diff
@@ -287,7 +287,7 @@ public String getCond(byte value) {
     }
 
     public RiseTransitSet getRTS(Alloc alloc) {
-        return new RiseTransitSet(schedule.getSite(), alloc.getObs().getCoords(), alloc.getStart());
+        return new RiseTransitSet(schedule.getSite(), alloc.getObs().getCoords(alloc.getVariant().getSchedule().getMiddlePoint()), alloc.getStart());
```

Comment:
Here we should be consistent and pass in the `Long -> Coordinates` function to `RiseTransitSet`.  In that class there is an `airmass` function that takes a time which you can apply to the function to get the precise coordinates.